cmd/admin-heal.go

```diff
@@ -29,23 +27,24 @@ import (
 	"github.com/minio/minio/pkg/madmin"
 )
 
-var (
-	adminHealFlags = []cli.Flag{
-		cli.BoolFlag{
-			Name:  "recursive, r",
-			Usage: "Heal recursively",
-		},
-		cli.BoolFlag{
-			Name:  "fake, k",
-			Usage: "Issue a fake heal operation",
-		},
-	}
-)
+var adminHealFlags = []cli.Flag{
+	cli.BoolFlag{
+		Name:  "recursive, r",
+		Usage: "Heal recursively",
+	},
+	cli.BoolFlag{
+		Name:  "dry-run, n",
```

Comment:
Examples are not updated, --fake is still being used.

Reply:
Fixed.